### DISCUSSION TITLE
Is it possible to create imagine larger than 1024x2?

### DISCUSSION THREAD

**Discussion by orionflame** [hlky/stable-diffusion#226] [Q&A]
Hi,

I saw that the UI sliders for width and height max out at 1K. Is it possible to create higher res without upres? MJ was allowing up to 2Kx2K.

Lastly is the aspect ratio working for any number like 7:12? I tried it, it still made a square pic. Not sure then how one is able to use both width and height and also AR.

Thanks a lot in advance.

**Comment by TingTingin** [comment]
After 1024x1024 the stable diffusion falls apart and starts generating random noise also mj doesn't allow 2Kx2K generation they upscale to that size just like this repo

**Comment by orionflame** [comment]
This guy claims to get larger images:
https://www.facebook.com/102903015822517/posts/pfbid0WtPjZsbjitvLXXS8YGxuuMJstrf9obSJudCEktXm3Fjda6jdceo68hSHztcxvxe2l/

Not sure if i am misreading.

**Comment by TingTingin** [comment]
One of the devs on discord said the max is 1024x1024 or one megapixel and after that all you get is a garbled mess you could extend on one axis only and do say 704x1472 though which would still be one megapixel

**Comment by orionflame** [comment]
That would also be cool but right now we can't do that using the web-ui. It should let us to either use width and height or AR with a specific width or height, but in either way the slider ranges should dynamically update to allow the valid range.

Thanks guys.

**Comment by TingTingin** [comment]
you can type "width 1536" to change width 1536 in the prompt box

**Comment by orionflame** [comment]
oh i didnt know this. i will try this then. so the prompt box overrides the parameters?

**Comment by cstueckrath** [comment]
the models are trained for 512x512. Therefore you'll get the best quality with this dimensions.
You can use AI upscaler (enhanced Real-ESRGAN) afterwards to increase the resolution

**Comment by TingTingin** [comment]
It doesn't override it automatically changes the actual value of the slider and the removes the text you will be able to see it visually change

**Comment by SafentisFox** [comment]
There's nothing stopping you from going into frontend/frontend.py and changing the maximum value allowed in txt2img_width to whatever you want.

Of course, the result you'll get out of it is probably not going to be great, considering that the model was never intended for such big resolutions. 

There has been some work by some people, getting the model to generate larger resolution images:
https://www.reddit.com/r/StableDiffusion/comments/x6dhks/fork_for_automatic_memory_allocation_allows_for/ who's using https://github.com/Doggettx/stable-diffusion which is based on https://github.com/neonsecret/stable-diffusion
